test/WebJobs.Script.Tests/TestScripts/CSharp/AssembliesFromSharedLocation/function.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bindings": [
```

Comment:
Because of the nature of what this does (importing the dependency at compilation), combining this with other scenarios might cause some issues. I'll pass on this for this PR but go through other tests to find good candidates for consolidation.